src/main/java/org/sagebionetworks/web/client/widget/FullWidthAlert.java

```diff
@@ -31,8 +31,15 @@ public FullWidthAlert() {
 	
 	private void rerender() {
 		Double autoCloseAfterDelayInSeconds = null;
-		AlertButtonConfig primaryButtonConfig = AlertButtonConfig.create(primaryButtonText, onPrimaryClick);
```

Comment:
Leave the configurations null if we don't specify text + an action (or disabled, for the secondary button)